docs/Makefile

```diff
@@ -49,6 +49,7 @@ node_modules:
 	npm config set registry http://registry.npmjs.org/
 	npm install --silent xml2js htmlparser imagemagick mocha@1.20.1 express@3.1.2 async html-to-text restler
 	mkdir -p node_modules
```

Comment:
I know you didn't add this line, but it's weird, npm install creates this so not sure why it's needed.